File: src/Entities/Medicine.Entities/Models/ActiveElement.cs
using Medicine.Entities.Models.Base;
using Medicine.Entities.Models.Translated;

namespace Medicine.Entities.Models
{
    public class ActiveElement : TranslationEntity<TranslatedActiveElement>
    {
        public int DrugId { get; set; }
        public Drug Drug { get; set; }
        public double Quantity { get; set; }
    }
}

File: front/medicine/src/app/shared/models/entity/ActiveElement.ts
import { User } from "./User";
import { TranslatedActiveElement } from "./translate/TranslatedActiveElement";
import { Drug } from "./Drug";


export class ActiveElement {
  drugId!: number;
  drug!: Drug;
  quantity!: number;
  translations: TranslatedActiveElement[] = [];
  user?: User;
  userId?: number;
  id!: number;
  createdAt?: Date;
}
